open-company/open-company-web
# Marketing site

Card: https://trello.com/c/dYkM5Yby/818-marketing-site

Port the ML design of the marketing site to our site and blog.

To test:
- navigate our marketing site starting from the home page and using internal navigation
- navigate our marketing site starting from the home page and using internal navigation with an iPhone (or enter device mode in chrome debug tools and select an iPhone model)
- navigate our marketing site starting from the home page and using internal navigation with an iPad (or enter device mode in chrome debug tools and select an iPad model)
- navigate to /about and /features and make sure they looks the same as you are coming from the homepage
- check the above from iphone and ipad
- navigate to blog
- check that the blog links are pointing where they should, they point to our production server so clicking will give you errors
- check the blog from iPhone and iPad
- check the blog post page with all 3 window sizes
- merge
- dance
- turn around and kiss your ferret (hope you removed the glands 🤣 )

This also fixes the fonts that were missing here https://github.com/open-company/open-company-web/pull/244#issuecomment-308690339
